setup.py

```diff
@@ -8,14 +8,14 @@
 import subprocess
 import sys
 from distutils import log as logger
```

Comment:
as said in the warning "make sure that setuptools is always imported before distutils." do we need update this distutils on line 10 as well?

Reply:
I looked into setuptools code to see how to replace it. I found setuptools is also using distutils log: https://github.com/pypa/setuptools/blob/main/setuptools/logging.py#L4. I moved distutils import.

Reply:
Since distutils is going to disappear, I replaced the logger by its implementation I found here: https://github.com/pypa/distutils/blob/main/distutils/_log.py.